src/generators/operationGenerator.ts

```diff
@@ -141,7 +143,7 @@ export function writeGetOperationOptions(
  * Generates a string representation of an Operation spec
  * the output is to be inserted in the Operation group file
  */
-function buildSpec(spec: OperationSpecDetails): string {
+function writeSpec(spec: OperationSpecDetails, writer: CodeBlockWriter) {
```

Comment:
can you add a return type to this function?

Reply:
:D good catch, this should be void. The return was a left over